batter/game/cast.py

```diff
@@ -11,6 +11,9 @@ def _setup_base_actors(self):
         self.setup_paddle()
         self.setup_bricks()
         self.setup_ball()
+        self.setup_top()
```

Comment:
We already have these being drawn by the output service. These aren't necessary.